sim/core/presim.go

```diff
@@ -65,6 +65,7 @@ func (sim *Simulation) runPresims(request proto.RaidSimRequest) {
 	presimRequest.SimOptions.Iterations = numPresimIterations
 	duration := DurationFromSeconds(presimRequest.Encounter.Duration)
 
+	var lastResult *proto.RaidSimResult
 	for remainingAgents > 0 {
```

Comment:
I'm pretty sure you need to modify this check so that the presim still runs once if health is selected but none of the agents need the presim themselves.

Reply:
good call